templates/index.html

```diff
@@ -38,390 +41,523 @@ <h2 class="p-heading--two">Say hello to your physical&nbsp;cloud</h2>
+      </li>
+      <li class="p-matrix__item">
+        <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}14939f84-devops-integration.svg" alt="DevOps image" />
+        <div class="p-matrix__content">
+          <h4 class="p-matrix__title"><span class="p-heading--five">Authentication & Identity</span></h4>
+          <div class="p-matrix__desc">
+            <p>Integrate with LDAP, Active Directory or SAML for central identity management and single-sign-on across multiple MAAS regions.</p>
           </div>
-          <div class="col-6">
-            <p>MAAS's API allows storage and networking to be rapidly reconfigured to match each test case, generating a clean bare metal environment for every run.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}c4778eb5-service-tracking.svg" alt="Service tracking image" />
+        <div class="p-matrix__content">
+          <h4 class="p-matrix__title"><span class="p-heading--five">Composable systems</span></h4>
+          <div class="p-matrix__desc">
+            <p>Drive Cisco UCS, Intel RSD, HP Moonshot and more. Supports dynamic hardware composition with Intel RSD.</p>
           </div>
         </div>
-        <div class="row">
-          <div class="col-2">
-            <h4 class="p-heading--five">Hardware testing at scale</h4>
+      </li>
+      <li class="p-matrix__item">
+        <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}c4778eb5-service-tracking.svg" alt="Service tracking image" />
+        <div class="p-matrix__content">
+          <h4 class="p-matrix__title"><span class="p-heading--five">Cloud metadata</span></h4>
+          <div class="p-matrix__desc">
+            <p>Reuse standard cloud operations with cloud-init and metadata services. Hybrid multicloud operations now include bare metal, with no change in applications.</p>
           </div>
-          <div class="col-6">
-            <p>Run sets of standard and customised tests on freshly provisioned machines and racks, transforming any new hardware provision and integration process.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}c4778eb5-service-tracking.svg" alt="Service tracking image" />
+        <div class="p-matrix__content">
+          <h4 class="p-matrix__title"><span class="p-heading--five">KVM Micro-cloud</span></h4>
+          <div class="p-matrix__desc">
+            <p>Designate servers to host KVM virtual machines to be dynamically provisioned alongside physical servers.</p>
           </div>
         </div>
-      </div>
+      </li>
+    </ul>
+  </div>
+</section>
+
+<section class="p-strip--accent is-shallow">
+  <div class="row u-vertically-center">
+    <div class="col-10">
+      <h3 class="p-heading--two u-min-margin--bottom">Learn how to configure and install MAAS</h3>
```

Comment:
Not a big fan of adding new spacing utilities here.

Reply:
Replaced with u-no-margin--bottom. I think @lyubomir-popov had it there to match line heights?